src/components/Hero/HeroContent.js

```diff
@@ -10,7 +10,7 @@ export default class HeroContent extends React.Component {
     const children = this.props.children;
     return (
       <div className={styles.content}>
-        {children ? children : ''}
```

Comment:
This code failed the updated airbnb eslint rules: [`unneeded-ternary`](http://eslint.org/docs/rules/no-unneeded-ternary.html).